cmd/iam.go

```diff
@@ -406,6 +406,7 @@ func (sys *IAMSys) periodicRoutines(ctx context.Context, baseInterval time.Durat
 	timer := time.NewTimer(waitInterval())
 	defer timer.Stop()
 
+	lastPurgeHour := 0
```

Comment:
```suggestion
	lastPurgeHour := -1
```

Right now if first refresh happens in the midnight hour (`refreshStart.Hour()=0`) then the purges will not happen until the next hour. This should make it so purges are always run on the first refresh after startup.